web/src/javascript/drawMap.js

```diff
@@ -34,9 +33,13 @@ export function createGameCanvas() {
   return document.createElement("canvas");
 }
 
-export function initMap(canvas) {
-  canvas.width = mapWidth;
-  canvas.height = mapHeight;
+export function initMap(canvas, numPlayers) {
+  canvas.width = constants.mapWidth*numPlayers;
```

Comment:
I don't think this logic should be on the frontend (how to calculate the size of the map), we can use the dimensions in `gameState.map` for the complete map size!